cpp/nodes_module/algorithm/nodes.cpp

```diff
@@ -2,12 +2,22 @@
 #include "mgp.hpp"
 
 namespace {
-void throw_exception(const mgp::Value &value) {
+void ThrowException(const mgp::Value &value) {
   std::ostringstream oss;
   oss << value.Type();
   throw mgp::ValueException("Unsuppported type for this operation, received type: " + oss.str());
 };
```

Comment:
Since there's only one point in code where `ThrowException` is called, perhaps we could do away with this function and move its body there.

Reply:
I think that function will also be used in other nodes procedures. Ideally I would put it some kind of utils (but I don't think the current structure uses that) or even cpp-api mgp::Value since its used very often.

Reply:
I see, thanks!